### PR TITLE
Add connector for uwu radio

### PR DESCRIPTION
**Pull request by redstonekasi**
**Describe the changes you made**
Adds a new connector for [uwu radio](https://radio.uwu.network/).

**Additional context**
image

### CODE REVIEW

File: src/connectors/uwu-radio.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+// If I understood correctly, these are all just backups in case the Media
+// Session API isn't supported.
+Connector.useMediaSessionApi();
```
Comment (inverse) [inline src/connectors/uwu-radio.js:5]: This is more for using the media session API for getting meta data about the playing music

**Review by inverse — COMMENTED** [review]
Looks like a good start - could leverage mediaSession for a lot of the work + some linting issues to fix :+1:

File: src/connectors/uwu-radio.js (hunk 1 above)
Comment (redstonekasi) [reply src/connectors/uwu-radio.js:3]: I made this connector before I added a polyfill for the Media Session API so it also works on older brothers, removed the other selectors now.

**Comment by redstonekasi** [comment]
Sorry I couldn't get the linter to run so I was just guessing the code style based on other files.

**Pull request merged**